terraformer_cloud_init.tf

```diff
@@ -0,0 +1,30 @@
+# cloud-init commands for configuring Terraformer instances
+
+data "cloudinit_config" "terraformer_cloud_init_tasks" {
+  gzip          = true
+  base64_encode = true
+
+  # Note: The filename parameters in each part below are only used to
+  # name the mime-parts of the user-data.  They do not affect the
+  # final name for the templates. For any x-shellscript parts, the
+  # filenames will also be used as a filename in the scripts
+  # directory.
+
+  # Create a configuration file for the VNC user that can be used to
+  # configure the AWS CLI to assume the Terraformer role by default.
+  # For details, see
+  # https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html#using-a-configuration-file
+  part {
+    content = templatefile(
+      "${path.module}/cloud-init/write-aws-config.tpl.yml", {
+        aws_region  = var.aws_region
+        owner       = "vnc:vnc"
+        path        = "/home/vnc/.aws/config"
```

Comment:
The VNC user's username should be pulled in from the SSM parameter store (see `var.ssm_key_vnc_username`).

Reply:
See commit f146058.

Reply:
That's the stuff right there! 🪐